### PR TITLE
Member Page for SCE members and non-member page/ message for non members

### PR DESCRIPTION
**Pull request by danknessdra**
This was done by changing the endpoint to give information to users with member membership status and above as opposed to officer. 

There is also a message for non members who want to access the member page.
nonmember:
image
member:
image

### CODE REVIEW

File: src/Pages/Profile/MemberView/Profile.js [hunk 1]
```diff
@@ -110,18 +110,18 @@ export default class Profile extends Component {
...
         { title: '', value: '' },
-        { title: 'Door Code', value: '' },
-        { title: 'Joined Date', value: '' },
-        { title: 'Email', value: '' },
-        { title: 'Membership Expiration', value: '' },
-        { title: '2D Prints', value: '' },
-        { title: 'Request 3D Printing', value: '' },
-        { title: 'Sign-in with Discord', value: '' },
-        { title: 'Change Password', value: '' }
+        { title: '', value: '' }
```
Comment (evanugarte) [inline src/Pages/Profile/MemberView/Profile.js:124]: instead of empty title and value can this array just be 1 element? or does it not work

**Review by evanugarte — APPROVED** [review]

File: src/Pages/Profile/MemberView/Profile.js (hunk 1 above)
Comment (danknessdra) [reply src/Pages/Profile/MemberView/Profile.js:124]: it fails the test if it is not 1 element

Comment (evanugarte) [reply src/Pages/Profile/MemberView/Profile.js:124]: maybe change the test instead? up to u i already approved

Comment (danknessdra) [reply src/Pages/Profile/MemberView/Profile.js:124]: sick bet

**Pull request closed without merging**